packages/cli/src/commands/record/record.command.ts

```diff
@@ -17,16 +17,19 @@ import { fetchAsset } from "../../local-data/replay-assets";
 import { getCommitSha } from "../../utils/commit-sha.utils";
 import { wrapHandler } from "../../utils/sentry.utils";
 
+// Note: We use `| undefined` instead of `?` here, despite yargs passing through absent keys for
+// unspecified options. We do this because recordCommandHandler is re-used by create-test
+// and we want to enforce that create-test passes through the full set of options, and does not forget any.
 export interface RecordCommandHandlerOptions {
-  apiToken?: string | null | undefined;
-  commitSha?: string | null | undefined;
-  devTools?: boolean | null | undefined;
-  bypassCSP?: boolean | null | undefined;
-  width?: number | null | undefined;
-  height?: number | null | undefined;
-  uploadIntervalMs?: number | null | undefined;
-  incognito?: boolean | null | undefined;
-  trace?: boolean | null | undefined;
+  apiToken: string | undefined;
+  commitSha: string | undefined;
+  devTools: boolean | undefined;
+  bypassCSP: boolean | undefined;
+  width: number | undefined | null;
+  height: number | undefined | null;
+  uploadIntervalMs: number | undefined | null;
+  incognito: boolean | undefined;
+  trace: boolean | undefined;
```

Comment:
yargs will only provide nulls in the case of numbers that don't pass e.g. `--width "I'm a string"`. For strings and booleans it'll only ever provide string/boolean or undefined (specifically absent key). So simplified our typings a bit to be more accurate.

Note that to be truly accurate (and so ensure correct usage of things like `{...spread}` operations), we'd want these as optional params:

```
apiToken?: string;
```

However in the cases where we have commands that call each other, and I want to make sure we don't accidentally forget to wire through new flags / the compiler forces us to think about all the cases when we add a new flag etc. Using `| undefined` instead of `?` is the way to force that.